Tests/NIOSSLTests/TLSConfigurationTest.swift

```diff
@@ -90,6 +93,9 @@ class TLSConfigurationTest: XCTestCase {
         (cert, key) = generateSelfSignedCert()
         TLSConfigurationTest.cert2 = cert
         TLSConfigurationTest.key2 = key
+
+        TLSConfigurationTest.compatibleSignatureAlgorithm = .rsaPssRsaeSha256
+        TLSConfigurationTest.incompatibleSignatureAlgorithm = .ecdsaSecp384R1Sha384
```

Comment:
These don't need to be at the test case scope, we can use them in the test directly.